dotcom-rendering/src/components/Lightbox.stories.tsx

```diff
@@ -324,7 +299,27 @@ export const WithLabs = () => {
 };
 
 /**
- * with html in caption (links)
- * with a portrait image
- *
+ * We toggle the [i] button to close and open the caption
  */
+export const ClickInfo = () => {
+	const format = {
+		display: ArticleDisplay.Standard,
+		design: ArticleDesign.Standard,
+		theme: Pillar.News,
+	};
+	const images = [{ ...testImage, title: 'ClickInfoTest' }];
+	return (
+		<Initialise>
+			<LightboxLayout format={format} images={images} />
+		</Initialise>
+	);
+};
+ClickInfo.play = async ({ canvasElement }: { canvasElement: HTMLElement }) => {
+	const canvas = within(canvasElement);
+
+	await userEvent.click(canvas.getByTitle('Toggle caption [i]'));
+	await expect(canvas.getByText(/ClickInfoTest/)).not.toBeVisible();
```

Comment:
To satisfy ESLint, `"include": ["./jest-setup.ts"]` is added to tsconfig

https://github.com/storybookjs/storybook/issues/21581#issuecomment-1493684531